src/plugins/intel_cpu/src/nodes/concat.cpp

```diff
@@ -391,7 +390,8 @@ void Concat::prepareParams() {
         if (canExecRef) {
             const auto srcMemDesc = srcMemPtr->getDescPtr()->as<BlockedMemoryDesc>();
             const auto& inputShape = srcMemDesc->getBlockDims();
-            inputStrides[i] = srcMemDesc->getStrides();
+            inputStrides[i].resize(MAX_RANK_REF, 0);
```

Comment:
Could you please explain what was the problem? We need to cover by tests I believe